sematic/api/endpoints/tests/test_runs.py

```diff
@@ -521,6 +532,19 @@ def test_update_future_states(
             "content": [{"future_state": "SCHEDULED", "run_id": persisted_run.id}]
         }
 
+        # Pretend the job disappeared
```

Comment:
Please extract this new code to a separate test case. Some code duplication is acceptable.